src/components/node-list/node-list-row-list.js

```diff
@@ -4,6 +4,19 @@ import { connect } from 'react-redux';
 import NodeListRow, { nodeListRowHeight } from './node-list-row';
 import LazyList from '../lazy-list';
 
+// modify display of labels for modular pipelines to show nested relationship
+const getItemLabel = (item) => {
```

Comment:
The rest of the label formatting is handled in selectors in `node-list-items.js` to improve performance. Do you think it would make more sense to handle this there?

Reply:
Yea I had a thought about this too, but given that this is more like a 'hack' rather than a permanent solution ( this indentation is subjected to further design changes) I believe this sits better here than in a selector ( so when the next design iteration happens this is more noticeable to spot as a temporary solution for modular pipelines). 

How about I add a comment to mention that this is a temporary formatting subject to delete / change on further design changes?

Reply:
Okay that's fine. The main reason I suggested this is as a performance improvement but it's not a big deal.